### DISCUSSION TITLE
[meta] Triaging old issues

### DISCUSSION THREAD

**Discussion by tkw1536** [nginx-proxy/nginx-proxy#1942] [General]
This repository has a lot of old issues - a bunch of these are outdated or fixed by now.
I've gone over the last five pages of the issue tracker (and plan to go over more pages, but that's enough for today). 

I identified a bunch of issues that could be closed or be turned into discussions.
I can't close them myself as I don't have the permissions.
Since there is a lot of these I figured I would post a list here instead of ccing the maintainers on each one to get them closed.

Can probably be closed:

- #73 (RTFM)
- #96 (seems to have been an issue with migrating from old versions, and fixed for new images by now)
- #101 (resolved by newer versions of `docker-gen`)
- #104 (`forego` issue is closed)
- #134 (closed in favor of #106)
- #171 (closed by #692)
- #203 (RTFM of docker)
- #234 (docker-compose issue is closed)
- #295 (solved by [`HTTPS_METHOD=noredirect`](https://github.com/nginx-proxy/nginx-proxy#how-ssl-support-works`))
- #305 (linked issue is closed)
- #355 (done automatically by GitHub Workflow)
- #360 (`docker-gen` issues are closed)
- #362 (resolved by #337)
- #380 (resolved in last comment)
- #389 (resolved in last comment)
- #394 (closed by #1607)
- #401 (duplicate of #133)
- #438 (resolved by recreating containers)
- #448 (RTFM)
- #458 (resolved in last comment)
- #461 (resolved in last comment)
- #468 (existing close request)
- #470 (RTFM)
- #490 (resolved in last comment)
- #497 (resolved in last comment)
- #500 (nginx is up-to-date now)
- #502 (resolved in last comment)
- #505 (existing close request, RTFM)
- #515 (fixed in #1184)
- #522 (resolved in last comment)
- #529 (answered in comments)
- #564 (existing close request)
- #570 (resolved in last comment)

Should probably be moved to discussions:

- #158
- #181
- #184
- #205
- #282
- #285
- #334
- #335
- #363
- #386
- #387
- #391
- #397
- #398
- #450
- #552
- #553
- #563
- #573

/cc @buchdag would appreciate if you could have a look at these.
I'm also open to closing them myself, if you are willing to give me Triage access to the repo.

**Comment by buchdag** [comment]
> I'm also open to closing them myself, if you are willing to give me Triage access to the repo.

Done, thank you for lending a hand 👍

**Reply by buchdag** [reply]
@tkw1536 Excellent work, thank you 👍  👍  👍 .

Let me know if you would also like Triage access over nginx-proxy/docker-gen and nginx-proxy/acme-companion (this one is pretty clean already).

**Reply by tkw1536** [reply]
Don't think it's necessary for now, although maybe later.
I've got plenty of issues still to cover on nginx-proxy itself.
